### PR TITLE
Update Office-365-groups.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4430
Added to note.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Office-365-groups.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ How Office 365 Groups work
...
+> Based on customer feedback, new Office 365 Groups generated as a result of creating a team in Microsoft Teams will no longer show in Outlook by default. For customers that want to continue with the existing behavior of showing these groups in Outlook, an Exchange Online PowerShell cmdlet will be provided which can enable the group for the Outlook experience. Groups created through Outlook and then later enabled for Teams will continue to show in both Outlook and Teams. This update will gradually roll out across Outlook and Teams in the coming months. If a new team is created via the Teams Admin Centre, or PowerShell, the team is not hidden from the GAL or Microsoft Outlook by default. Run the PowerShell command 'HiddenFromExchangeClientsEnabled' for this to hide the team from the Global Address List and Outlook clients. See here [Set-UnifiedGroup](https://docs.microsoft.com/powershell/module/exchange/users-and-groups/set-unifiedgroup?view=exchange-ps).
```
Comment (JohanFreelancer9) [inline Teams/Office-365-groups.md:41]:
```suggestion
> Based on customer feedback, new Office 365 Groups generated as a result of creating a team in Microsoft Teams will no longer show in Outlook by default. For customers that want to continue with the existing behavior of showing these groups in Outlook, an Exchange Online PowerShell cmdlet will be provided which can enable the group for the Outlook experience. Groups created through Outlook and then later enabled for Teams will continue to show in both Outlook and Teams. This update will gradually roll out across Outlook and Teams in the coming months. If a new team is created via the Teams admin center or PowerShell, the team is not hidden from the GAL or Microsoft Outlook by default. Run the PowerShell command 'HiddenFromExchangeClientsEnabled' for this to hide the team from the Global Address List and Outlook clients. See [Set-UnifiedGroup](https://docs.microsoft.com/powershell/module/exchange/users-and-groups/set-unifiedgroup?view=exchange-ps) for details.
```

Comment (JohanFreelancer9) [reply Teams/Office-365-groups.md:41]: 1. Change "Teams Admin Centre" to "Teams admin center" - please note the spelling of "center".
2. Remove the comma after "center".
3. Change the last sentence as indicated.

Comment (JohanFreelancer9) [inline Teams/Office-365-groups.md]:
```suggestion
> Based on customer feedback, new Office 365 Groups generated as a result of creating a team in Microsoft Teams will no longer show in Outlook by default. For customers that want to continue with the existing behavior of showing these groups in Outlook, an Exchange Online PowerShell cmdlet will be provided which can enable the group for the Outlook experience. Groups created through Outlook and then later enabled for Teams will continue to show in both Outlook and Teams. This update will gradually roll out across Outlook and Teams in the coming months. If a new team is created via the Teams Admin Centre, or PowerShell, the team is not hidden from the GAL or Microsoft Outlook by default. Run the PowerShell command 'HiddenFromExchangeClientsEnabled' for this to hide the team from the Global Address List and Outlook clients. See [Set-UnifiedGroup](https://docs.microsoft.com/powershell/module/exchange/users-and-groups/set-unifiedgroup?view=exchange-ps) for details.
```

Comment (JohanFreelancer9) [reply Teams/Office-365-groups.md]: Please change last sentence as per edit.

Comment (lindspea) [reply Teams/Office-365-groups.md]: Hi @JohanFreelancer9  it's showing that it is changed on my side. Please double check. Are you referring to adding "for details" at the end? It is showing on my side.

Comment (JohanFreelancer9) [reply Teams/Office-365-groups.md]: Hi @lindspea  Yes it is fine now! Thanks.

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
@abgupta please review this PR. LMK if it's OK to merge.

**Comment by LolaJacobsen** [comment]
Per PM, don't merge. There are changes coming.

**Pull request closed without merging**